yarn.lock

```diff
@@ -2460,6 +2472,73 @@ ast-types@^0.13.2:
   dependencies:
     tslib "^2.0.1"
 
+astro-icon@^0.5.3:
+  version "0.5.3"
+  resolved "https://registry.yarnpkg.com/astro-icon/-/astro-icon-0.5.3.tgz#095b6be506a2a021ef7e6ce0cce83e2699d46862"
+  integrity sha512-Tt+w6OyuE4NlvP1LjT09L8JD23+j9U2LYqEOtPDr7gDJXAZxa4mwz7pOJo9RNhdRx1SXh2aS8KK9v7D2N5d0aw==
+  dependencies:
+    node-fetch "^3.1.0"
+    svgo "^2.8.0"
+
+astro@0.23.0-next.6:
```

Comment:
This might be the culprit. Not in sync with our local version.

Reply:
Oh good catch, I'll rebase.